### ISSUE TITLE
Recipe from Craftable Weapons Pack for removed .22 and .38 Luty SMGs  causing error messages

### ISSUE THREAD

**Issue by LaVeyanFiend** [CleverRaven/Cataclysm-DDA#33667]
# Describe the bug

Getting invalid crafting result message on loading world with the Craftable Guns mod due to removal of .22 and .38 Luty SMGs in #33376.

# Steps To Reproduce

1. Create world with Craftable Guns Pack.
2. Make character on world.
3. Receive error in screenshot.

# Expected behavior

No error message.

# Screenshots

![image](https://user-images.githubusercontent.com/51099123/63985075-f1623e80-ca9b-11e9-8a76-9ce9128b4740.png)

# Versions and configuration

All builds after Experimental 9552, CDDA and Craftable Weapons Pack.

# Additional context

I'd make the fix myself, but GitKraken's giving me errors with the new PR validator blocking pushing to my remote for PRing. Will write fix once #33666 is merged if someone else doesn't do it first.

**Comment by AMurkin** [comment]
> Will write fix once #33666 is merged if someone else doesn't do it first.

Wrong PR number?

**Comment by tenmillimaster** [comment]
Should the fix be deleting those recipes, or adding them back to the mod?

**Comment by kevingranade** [comment]
You can obsolete the recipes, see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/recipe_obsolete.json
